tests/console/ssh_cleanup.pm

```diff
@@ -0,0 +1,21 @@
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use base "consoletest";
+use strict;
+use testapi;
+
+# Summary: Cleanup ssh test user to prevent the user showing up in
+#  displaymanager and confusing other tests
+# Maintainer: okurz@suse.de
+sub run() {
+    select_console 'root-console';
```

Comment:
Is this really needed ? this tests are always in console or ?
https://openqa.suse.de/tests/593483#step/ssh_pubkey/5